bad1/bad1Data/mod/track_redirect.mod

```diff
@@ -422,6 +422,142 @@ MOD_REL(
         b _battle_id_to_track_id;
 )
 
+
+/* These mods add proper CT-CODE logic to the "In Order" and "Random" VS settings.
+ * Prior to a VS game, a pool of 32 course IDs is generated, which determines what
+ * course is played next upon selecting "Next Course". Not only do we need to make
+ * custom tracks poolable (rather than just their special slot IDs), we must also
+ * redirect the "Next Course" handler to take advantage of CT-CODE.
+ * 
+ * For reference: "Random" picks the 1st course, pools 31 more in subsequent order,
+ * then randomly permutes each pooled course with another. Consequently, on CT-CODE,
+ * courses that are 32+ slots apart from each other are never pooled together.
+ */
+MOD_REL(
+    mod_redirect_vs_1,
+    mod_redirect_vs_1_addr,
+        /* Reload current track ID into param r4. "In Order" passes us its slot instead. */
+        lis r6, raceCupTable@ha;
+        lwz r4, currentCourse@l(r6);

```

Comment:
Now that I think about it, the mods that redirect ```raceCupTable``` (vs_1 twice, vs_6, vs_7) could be isolated, renamed, and moved up into the section of the file that groups all the ```mod_raceCupTable``` mods together.
Once that's done, all that would be left from this mod (besides redirection) is this particular instruction. Heck, it wouldn't be needed at all, if not for one of the callees of this function passing into r4 a special slot ID that's "returned" by ```redirect_course```.